kitty-items-cadence/contracts/FungibleToken.cdc

```diff
@@ -0,0 +1,199 @@
+/**
```

Comment:
Could we import this from the Flow fungible token repo instead of copying it here?